edx_exams/apps/router/middleware.py

```diff
@@ -0,0 +1,32 @@
+"""
+Middleware that checks if in incoming request should be
+routed to the legacy proctoring service (edx-proctoring)
```

Comment:
maybe this class and the map and so on should be named legacy or proctoring? Since this is not a generic override, it's the ExamRequestLegacyRedirectMiddleware. Except that's too long.